drivers/hmis/app/models/hmis/form/assessment_processor.rb

```diff
@@ -45,7 +48,17 @@ def run!
   # Type Factories
   def enrollment_factory(create: true) # rubocop:disable Lint/UnusedMethodArgument
     # The enrollment has already been created, so we can just return it
-    assessment_detail.assessment.enrollment
+    enrollment = assessment_detail.assessment.enrollment
+    enrollment
```

Comment:
Is this assignment just to ease debugging?

Reply:
Oh I added and then removed some code - I will remove the assignment